src/V3FileLine.cpp

```diff
@@ -360,12 +361,16 @@ void FileLine::v3errorEnd(std::ostringstream& sstr, const string& locationStr) {
         lstr << std::setw(ascii().length()) << " "
              << ": " << locationStr;
     }
-    if (warnIsOff(V3Error::errorCode())
-        || V3Config::waive(this, V3Error::errorCode(), sstr.str())) {
+    bool waive = V3Config::waive(this, V3Error::errorCode(), sstr.str());
+
+    if (warnIsOff(V3Error::errorCode()) || waive) {
```

Comment:
You have this in the other pull request, so either remove it from this, or put it in here and close the other pull please.

Reply:
oh, yeah, I first want to wait for #2355 to get merged and then rebase on top of that, so keeping this open for resolution once the other one is merged. The other one came up during testing, so things get a bit mixed at this line of code.